handlers/config.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("node:fs");
+const yaml = require("js-yaml");
+const cliColor = require("cli-color");
+
+console.log(cliColor.cyanBright("[PteroStats] ") + cliColor.yellow("Loading configuration..."))
+
+let config = yaml.load(fs.readFileSync("./config.yml", "utf8"));
+if (fs.existsSync("config-dev.yml")) {
+    console.log(cliColor.cyanBright("[PteroStats] ") + cliColor.yellow("Using development configuration..."))
+    config = yaml.load(fs.readFileSync("./config-dev.yml", "utf8"));
+}
+
+try {
+    const testURL = new URL(process.env?.PanelURL);
+    if (!testURL.protocol.startsWith("http")) throw new Error();
+} catch {
+    console.error('Config Error | Invalid URL Format! Example Correct URL: "https://panel.example.com"');
+    process.exit();
+}
```

Comment:
If user input invalid URL, for example `example.com` it will return error and crash the bot before it can check the protocol, that's why I use try catch